### PR TITLE
PULL_REQUEST_TEMPLATE.md: cobot --> corobo

### PR DESCRIPTION
**Pull request by nguptaa**
PULL_REQUEST_TEMPLATE.md: cobot --> corobo 

Closes https://github.com/coala/documentation/issues/503

### CODE REVIEW

**Comment by Makman2** [comment]
> Please don't close PRs if your commits are not ready yet, you can always fix them up in the same PR.

**Comment by Makman2** [comment]
Please fix your commits up here.

**Comment by Makman2** [comment]
> Sorry I tried but couldn't figure it out. That's why I closed the PR, deleted repo and cloned the repo once again.

That's bad, please never do again :) Please read up our docs, it describes how to fix up your PRs.

**Comment by nguptaa** [comment]
Sure @Makman2 This won't repeat again :)

File: PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,9 +1,4 @@
-<!--
-Thanks for your contribution!
 
-Please take a quick look at those things down there. They're quite important.
-Really! We wrote them for you. Yes you! With utmost care. Read them.
--->
```
Comment (Makman2) [inline PULL_REQUEST_TEMPLATE.md]: Not part of the issue.

**Comment by Makman2** [comment]
Please read our commit message guidelines. Also you should clean up your commit body ;)

**Comment by Makman2** [comment]
unack a05640f285fb2d26c8450be9c8d943580d72f773

**Comment by sangamcse** [comment]
Use `git commit --amend` to edit commit message.

**Comment by ishanSrt** [comment]
You have written the same sentence twice in commit body. No need for that.
Second this is not a typo. Typos are mistakes while typing while this is due to the death of cobot and birth of corobo

**Comment by ishanSrt** [comment]
So change your shortlog and body, also full stop missing in commit body

**Comment by sangamcse** [comment]
Why is `gitmate` not catching the commit issue??

**Comment by ishanSrt** [comment]
Redundant commit body.

**Comment by ishanSrt** [comment]
@Vedarth the commit is perfectly fine. Please go through the link you have provided again if you are missing something 👍

File: PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -1,10 +1,3 @@
-<!--
-Thanks for your contribution!
-
-Please take a quick look at those things down there. They're quite important.
-Really! We wrote them for you. Yes you! With utmost care. Read them.
--->
-
```
Comment (ishanSrt) [inline PULL_REQUEST_TEMPLATE.md]: why this deletion? is that specified in the issue?

Comment (nguptaa) [reply PULL_REQUEST_TEMPLATE.md]: No. But after i changed cobot-->corobo this automatically got added.

Comment (ishanSrt) [reply PULL_REQUEST_TEMPLATE.md]: that's not possible

Comment (ishanSrt) [reply PULL_REQUEST_TEMPLATE.md]: maybe your branch is behind coala master

Comment (ishanSrt) [reply PULL_REQUEST_TEMPLATE.md]: sync your branch with coala master

**Comment by sangamcse** [comment]
You need to squash your commit. Use `git reset --hard 09233f25e619230758c033040cff911277178a95` and then `git commit --amend` to write commit message and `git push -f`.

**Comment by sangamcse** [comment]
Now, whenever you do changes on a PR, first do changes in file and then use `git add .` and then `git commit --amend`. After that `git push -f`.

**Comment by nguptaa** [comment]
@sangamcse  Thanks, I will follow it :)

**Comment by Makman2** [comment]
ack b5e896104f03a6f4a6c54b4ac6703644ad611952

**Comment by jayvdb** [comment]
@gitmate-bot rebase

**Comment by jayvdb** [comment]
reack https://github.com/coala/documentation/pull/533/commits/6002139fb5389a65610840f3ef09241f351a558b

**Comment by jayvdb** [comment]
@gitmate-bot ff

**Pull request merged**